src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -39,17 +39,17 @@ public static int Run(string[] args)
                 CommandOptionType.SingleValue);
 
             var listTestsOption = cmd.Option(
-                "-lt|--listTests",
+                "-t|--list-tests",
```

Comment:
`-l` is also used for `--language` in `dotnet-new`. I don't think it's something we need to account for since both are one-offs, but worth mentioning.